### PR TITLE
Updates to mergeduty after 71.0 cycle

### PR DESCRIPTION
**Pull request by MihaiTabara**

### CODE REVIEW

File: docs/mergeduty/howto.md [hunk 1]
```diff
@@ -25,20 +25,21 @@ How are those repositories kept in sync? That's `MergeDuty` and is part of the `
   * [Reply migrations are complete](#reply-to-relman-migrations-are-complete)
 * A week after Merge day, bump mozilla-central:
   * [Merge central to beta](#merge-central-to-beta)
-  * [Bump mozilla-esr](#bump-esr-version)
+  * [Re-open trees](#re-opening-the-trees)
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: good call

File: docs/mergeduty/howto.md [hunk 2]
```diff
@@ -184,7 +185,7 @@ Run the bump-esr [no-op trial run](#do-migration-no-op-trial-runs), and show the
 
 ```sh
 export version=68
-export full_version=68.3.0
+export full_version=68.4.0
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: maybe we should template this? Rather than bump *docs* each time?

File: docs/mergeduty/howto.md [hunk 3]
```diff
@@ -274,18 +275,17 @@ python mozharness/scripts/merge_day/gecko_migration.py \
 
 :warning: The decision task of the resulting pushlog in the `mozilla-beta` might fail in the first place with a timeout. A rerun might solve the problem which can be caused by an unlucky slow instance.
 
+### Re-opening the tree(s)
+
+kalk to Sheriffs and RelMan to re-open trees (ether `open` or `approval-only`) so that l10n bumper can run.
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: tiny nits

s/kalk to/ask/ ?

s/ether/either

File: docs/mergeduty/howto.md (hunk 2 above)
Comment (MihaiTabara) [reply docs/mergeduty/howto.md]: Thought of it too, but I was also hoping that by the next time we perform mergeduty, we may have some progress towards automation, hence that'll be coming from tree or so.

File: docs/mergeduty/howto.md (hunk 3 above)
Comment (MihaiTabara) [reply docs/mergeduty/howto.md]: Good catch, fixed both in a follow-up.

**Pull request merged**